snapshot_manager/tests/github_util_test.py

```diff
@@ -481,7 +486,7 @@ def test_remove_labels_safe(issue_mock: mock.Mock):
     )
 
 
-def test_minimize_comment_as_outdated__with_issue_comment(github_client_fxt):
+def test_minimize_comment_as_outdated__with_issue_comment(github_client_fxt) -> None:  # type: ignore[no-untyped-def]
```

Comment:
When using a pytest fixture it seems a known problem that mypy complains and annotating the fixture correctly I don't know how to do that. I also didn't want to skip checking the test code which would have been the other option.

Reply:
I didn't understand this message. Are you saying the `# type: ignore[no-untyped-def]` annotations were added automatically?
Can they be replace by the right types?

Reply:
No no, when you don't have the comment at the end of the line, mypy complains with i.e.:

```
Function is missing a type annotation for one or more argumentsMypy[no-untyped-def](https://mypy.readthedocs.io/en/latest/_refs.html#code-no-untyped-def)
```

That's why I've added the comment, to silence this complaint.

Reply:
Why don't you specify the type of the parameter? e.g.
```
def test_minimize_comment_as_outdated__with_issue_comment(github_client_fxt: github_util.GithubClient) -> None:
```
Does mypy also complain about that?